token-account-based/chaincode-go/chaincode/token_contract.go

```diff
@@ -200,6 +209,14 @@ func (s *SmartContract) Approve(ctx contractapi.TransactionContextInterface, spe
 		return fmt.Errorf("failed to update state of smart contract for key %s: %v", allowanceKey, err)
 	}
 
+	// Emit the Approval event
+	var approvalEvent = map[string]string{"owner": owner, "spender": spender, "value": strconv.Itoa(value)}
```

Comment:
nit: Objects with known values are generally defined using `:=` unless the object is in the global scope
```
approvalEvent := map[string]string{"owner": owner, "spender": spender, "value": strconv.Itoa(value)}
```